README.md

```diff
@@ -1,9 +1,25 @@
-# ARCHILATOR
+# 🅰🆁🅲🅷🅸🅻🅰🆃🅾🆁
 
-A simple terminal-based calculator
+A simple terminal-based calculator 🧮
 
-### Dependencies
+## Dependencies
+  - [node.js](nodejs.org)
+  - [npm](https://www.npmjs.com/)
 
-### Setup
+## Setup  
+1. Download the .zip file or clone the repo using: 
```

Comment:
nit: `using the github command line tool`

Reply:
added